qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/VisitNoteSteps.java

```diff
@@ -102,12 +99,12 @@ public void userclicksOnEditVisitNote() {
     public void editVisitNote() {
         visitNotePage.removeDiagnosis();
         visitNotePage.editNote();
-        visitNotePage.addSecondaryDiagnosis(DIAGNOSIS_SECONDARY_UPDATED);
+        visitNotePage.addSecondaryDiagnosis("Pneumonia");
```

Comment:
Can you explain why you needed to replace the `DIAGNOSIS_SECONDARY_UPDATED` constant?

Reply:
These changes happened when the server was at peek that it could not load most of the files appropriately which resulted them  to break. Howeverm Let me revert them back, these elements keep breaking due to unstability of our servers.